doc/faq.rst

```diff
@@ -186,26 +187,53 @@ No, you can use symbolic names for messages::
 
 
 4.5 I have a callback function where I have no control over received arguments. How do I avoid getting unused argument warnings?
-----------------------------------------------------------------------------------------------------------------------------------
+--------------------------------------------------------------------------------------------------------------------------------
 
 Prefix (ui) the callback's name by `cb_`, as in cb_onclick(...). By
 doing so arguments usage won't be checked. Another solution is to
 use one of the names defined in the "dummy-variables" configuration
 variable for unused argument ("_" and "dummy" by default).
 
 4.6 What is the format of the configuration file?
----------------------------------------------------
+-------------------------------------------------
 
 Pylint uses ConfigParser from the standard library to parse the configuration
 file.  It means that if you need to disable a lot of messages, you can use
-tricks like: ::
+any formatting accepted by ConfigParser, e.g.
+
+.. code-block:: ini
+
+    [MASTER]
+    output-format = colorized
+
+    [Messages Control]
+    disable=method-hidden,too-many-lines,wildcard-import
+
+.. code-block:: ini
 
-    # disable wildcard-import, method-hidden and too-many-lines
-    disable=
-        wildcard-import,
-        method-hidden,
+    [Messages Control]
+    disable =
+        method-hidden
         too-many-lines
+        wildcard-import
+
+Alternatively, if you use ``pyproject.toml``, e.g.
+
+.. code-block:: toml
+
+    [tool.pylint.master]
+    output-format = "colorized"
+
+    [tool.pylint.messages_control]
+    disable = [
+        "method-hidden",
+        "too-many-lines",
+        "wildcard-import",
+    ]
+
+    See also the `exhaustive list of possible options`_
 
+.. _`exhaustive list of possible options`: https://pylint.pycqa.org/en/latest/user_guide/configuration/all-options.html
```

Comment:
I'd better make this a local link. :worried:

Reply:
It's a generated file I think, do you know how we could make a local link ?

Reply:
Since links are checked after extensions are loaded I think this should be able to directly link to the generated file.

Reply:
@Pierre-Sassoulas I'll try to make it work [as described in the Sphinx docs](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#automatically-label-sections).